docs/rules/bugs/argument-always-wildcard.md

```diff
@@ -0,0 +1,92 @@
+# argument-always-wildcard
+
+**Summary**: Argument is always a wildcard
+
+**Category**: Bugs
+
+**Avoid**
+```rego
+package policy
+
+import rego.v1
+
+last_name(name, _) := lname if {
```

Comment:
would a clearer example be a function with two heads?

Reply:
Or a comment to explain it's the end of the file after the single head.

Reply:
It's provided later in the doc, but I'll add a comment above this to clarify